k8s/demo/fio/demo-cstor-limits-sc.yaml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Can we put some top level comments that explains in details the cas config, other properties, what it does etc.
This is optional, but can be done eventually.

Reply:
Will take this as part of 0.8 release docs.. some more refactoring will be done with these YAMLs.